javascript/core/src/verbs/util/boolean-logic.ts

```diff
@@ -87,25 +89,28 @@ export function and(comparisons: (1 | 0 | null)[]): 1 | 0 | null {
  * @returns
  */
 export function xor(comparisons: (1 | 0 | null)[]): 1 | 0 | null {
```

Comment:
Should we replace the bulk of this function with the even/odd logic to simplify?